x/sequencer/keeper/invariants.go

```diff
@@ -85,23 +85,21 @@ func SequencersPerRollappInvariant(k Keeper) sdk.Invariant {
 				continue
 			}
 
-			// FIXME: check for active proposer
-			/*
-				proposerFound := false
-				for _, seq := range bonded {
-					if seq.Status == types.Proposer {
-						if proposerFound {
-							broken = true
-							msg += "more than one proposer for rollapp " + rollapp.RollappId + "\n"
-						}
-						proposerFound = true
+			proposerFound := false
+			for _, seq := range bonded {
+				if seq.Proposer {
+					if proposerFound {
+						broken = true
+						msg += "more than one proposer for rollapp " + rollapp.RollappId + "\n"
 					}
+					proposerFound = true
 				}
-				if !proposerFound {
-					broken = true
-					msg += "no proposer for rollapp " + rollapp.RollappId + "\n"
-				}
-			*/
+			}
+			if !proposerFound {
```

Comment:
not sure it's a invariant break. we can have a frozen rollapp or a rollapp with no active sequencer which is waiting for a sequencer to come up. we can't force a rollapp to have a proposer - e.g if a proposer is unbonding and don't want to propose anymore or it's jailed.

Reply:
we have 
```
if len(bonded) == 0 {
				continue
			}
			
			
So if there are bonded sequencers, i would expect a proposer one